Specs/Number/Arabic/NumberModel.json

```diff
@@ -2026,27 +2026,27 @@
         "TypeName": "number",
         "Resolution": {
           "subtype": "fraction",
-          "value": "0.047619048"
+          "value": "0.0476190476190476"
         },
         "Start": 0,
         "End": 17
       }
     ]
   },
   {
-    "Input": "مائة ألف أخماس",
+    "Input": "مائة على ألف وخمسة",
```

Comment:
What was the previous input if not a fraction? Could we add that form to the specs for one of the other types?

Reply:
The input was intended to be fraction but we figured that out there was some discrepancy so we reviewed with Language Expert, and got corrected them. Older form of input was not valid native sentence. That was integer (but still not in correct form). 
We have such integer cases.